src/cart/crt_init.c

```diff
@@ -145,6 +146,12 @@ static int data_init(int server, crt_init_options_t *opt)
 		setenv("FI_UNIVERSE_SIZE", "2048", 1);
 	}
 
+	d_getenv_int("CRT_MRC_ENABLE", &mrc_enable);
+	if (mrc_enable == 0) {
+		D_INFO("Disabling MR CACHE (FI_MR_CACHE_COUNT=0)\n");
+		setenv("FI_MR_CACHE_COUNT", "0", 1);
```

Comment:
@frostedcmos Shouldn't this be FI_MR_CACHE_MAX_COUNT instead of FI_MR_CACHE_COUNT?

Reply:
oops. good catch. i've submitted new pr to fix it https://github.com/daos-stack/daos/pull/4637